images/windows/toolsets/toolset-2025.json

```diff
@@ -0,0 +1,328 @@
+            "versions": [
+                "3.9.*",
+                "3.10.*",
+                "3.11.*",
+                "3.12.*",
+                "3.13.*"
+            ],
+            "default": "3.9.*"
+        },
+        {
+            "name": "PyPy",
+            "arch": "x86",
+            "platform" : "win64",
+            "versions": [
+                "3.9",
+                "3.10"
+            ]
+        },
+        {
+            "name": "node",
+            "url" : "https://raw.githubusercontent.com/actions/node-versions/main/versions-manifest.json",
+            "arch": "x64",
+            "platform" : "win32",
+            "versions": [
+                "18.*",
+                "20.*",
+                "22.*"
+            ]
+        },
+        {
+            "name": "go",
+            "url" : "https://raw.githubusercontent.com/actions/go-versions/main/versions-manifest.json",
+            "arch": "x64",
+            "platform" : "win32",
+            "versions": [
+                "1.21.*",
+                "1.22.*",
+                "1.23.*"
+            ],
+            "default": "1.23.*"
+        }
+    ],
+    "powershellModules": [
+        { "name": "DockerMsftProvider" },
+        { "name": "MarkdownPS" },
+        { "name": "Pester" },
+        { "name": "PowerShellGet" },
+        { "name": "PSScriptAnalyzer" },
+        { "name": "PSWindowsUpdate" },
+        { "name": "SqlServer" },
+        { "name": "VSSetup" },
+        { "name": "Microsoft.Graph" },
+        {"name": "AWSPowershell"}
+    ],
+    "azureModules": [
+        {
+            "name": "az",
+            "versions": [
+                "12.4.0"
+            ],
+            "zip_versions": []
+        }
+    ],
+    "java": {
+        "default": "17",
+        "versions": [ "8", "11", "17", "21"]
+    },
+    "android": {
+        "commandline_tools_url": "https://dl.google.com/android/repository/commandlinetools-win-12266719_latest.zip",
+        "hash": "F9088C04A44F1F37A8A3A228A7663E11AE9445FA07529C96CEF38ACB985A88F3",
+        "platform_min_version": "35",
+        "build_tools_min_version": "35.0.0",
+        "extras": [
+            "android;m2repository",
+            "google;m2repository",
+            "google;google_play_services"
+        ],
+        "addons": [],
+        "additional_tools": [
+            "cmake;3.22.1",
+            "cmake;3.30.5"
+        ],
+        "ndk": {
+            "default": "27",
+            "versions": [
+                "26", "27"
+            ]
+        }
+    },
+    "mingw": {
+        "version": "14.2.0",
+        "runtime": "ucrt"
+    },
+    "MsysPackages": {
+        "msys2": [],
+        "mingw": []
+    },
+    "windowsFeatures": [
+        { "name": "Containers" },
+        { "name": "Microsoft-Windows-Subsystem-Linux", "optionalFeature": true },
+        { "name": "VirtualMachinePlatform", "optionalFeature": true },
+        { "name": "NET-Framework-45-Features", "includeAllSubFeatures": true },
```

Comment:
Why is .NET framework 3.5.x not in the features list?

Reply:
It never was there, we install .Net 3.5 [using the "Add Roles and Features" Wizard](https://learn.microsoft.com/en-us/windows-hardware/manufacture/desktop/enable-net-framework-35-by-using-the-add-roles-and-features-wizard?view=windows-11). The feature name - `NET-Framework-Features`